src/common/hooks/useFieldErrorMsg.tsx

```diff
@@ -23,7 +23,10 @@ const useFieldErrorMsg = ({ name, inputError, validate }: Props): string | undef
     }
   }, [field.value, setHadValue]);
   const showError = field.value || (!field.value && hadValue);
-  const errorMessage = (showError ? error : '') || inputError;
+  let errorMessage = (showError ? error : '') || inputError;
```

Comment:
Why not just do this?
```suggestion
  let errorMessage = showError ? error : inputError;
```

Reply:
This is code from before, but sure, I'll change it

Reply:
I don't think the code is equivalent so it should not be changed.
If  `inputError` is truthy, it's used as a fallback if there is no error or we don't want to display it.
This only seems to be used in `BMCForm` but I think it would affect it.